docs/api/tasks-delete-task

```diff
@@ -0,0 +1,47 @@
+layout: page
+---
+
+# delete task
+
+Delete a task from the to-do service. 
+
+## URL
+
+```shell
+
+{DELETE}{server_url}/tasks/
+```
+
+## Request headers
+
+This request does not use any authorization. The endpoint is available to all users and applications.
+
+| Header name | Description | Required | Values |
+| -------------- | ------ | ------------ |------------ |
+| Task | The task to be deleted. | Optional | application/json. Default value.  || 
+
+## Request body
+
+In the request body, specify a JSON representation of the [`user`](user) object. 
```

Comment:
```suggestion
None
```

I don't think you need  a request body for this operation.
See `api/users-get-user-by-id.md` for another example of such an operation.